gen/dcos-config.yaml

```diff
@@ -315,6 +315,11 @@ package:
       MESOS_GC_DELAY={{ gc_delay }}
       MESOS_HOSTNAME_LOOKUP=false
       GLOG_drop_log_memory=false
+  - path: /etc/dcos-log-config.json
+    content: |
+      {
+        "verbose": true
```

Comment:
the reason for config here is to be consistent with DC/OS EE. https://github.com/mesosphere/dcos-enterprise/pull/407
in this case i don't need to diverge the systemd unit and will only have to bump the upstream for this service to start.